### PR TITLE
London Class 7 - Joy Omodiale - Git/GitHub - Week 2

### PR DESCRIPTION
**Pull request by jmodiale**
Corrected the answers

<!-

# Your Details

- Your Name: Joy Omodiale
- Your City: London
- Your Slack Name: Joy Omodiale

# Homework Details

- Module: Git/GitHub
- Week: 2

### CODE REVIEW

**Review by Hozan94 — COMMENTED** [review]
Hi @jmodiale , I noticed you havent fixed the errors in some of the questions (7, 11, 12), let me know if you need any help?

**Comment by jmodiale** [comment]
Thanks, Hozan, I think I've made the corrections

File: homework.md [hunk 1]
```diff
@@ -4,38 +4,38 @@
...
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website beautiful & mobile friendly
```
Comment (ASLANDENIZ) [inline homework.md:31]: I like your answer, only you might  want to add; it allows one to adapt the presentation to different types of devices, such as large screens, small screens, or printers.

**Review by ASLANDENIZ — COMMENTED** [review]
I finished my review, it looks good, well done!

**Pull request closed without merging**